app/views/users/show.html.erb

```diff
@@ -1,2 +1,32 @@
-<h1>Details about one user will be displayed here</h1>
-<p>This is the page for displaying user info</p>
\ No newline at end of file
+<div class="card" style="width: 30rem;">
+  <div class="user-details container fluid d-flex flex-row border-bottom">
+    <img src='<%= @user.photo %>' class="card-img-top" id="user-photo" alt="walk">
+    <div class="card-body d-flex flex-column">
+      <h3 class="card-title"><%= @user.name %></h3>
+      <p class="card-text text-end"><small class="text-body-secondary">Number of posts: <%= @user.posts_counter %></small></p>
+    </div>
+  </div>
+  <div class="card-body">
+    <h5><strong>Bio</strong>:</h5>
+    <p><%= @user.bio %></p>
+  </div>
+
+  <ul class="list-group list-group-flush">
+    <% i=1 %>
+    <% @user.most_recent_three_posts.each do |p| %>     
+      <li class="list-group-item">
+        <h5>Post <%= i %>: <strong><%= p.title %></strong></h5>
+      </li>
```

Comment:
- [ ] It would be more user friendly if you can wrap up the three most recent post with a link tag to the `post_show` page as you rightly did to `view post` in `post_index` page. ℹ️